integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java

```diff
@@ -726,21 +729,37 @@ void testUpdateImageName() {
     String imageUpdate = KIND_REPO != null ? KIND_REPO
         + (WEBLOGIC_IMAGE_NAME + ":" + imageTag).substring(TestConstants.BASE_IMAGES_REPO.length() + 1)
         : TEST_IMAGES_REPO + "/" + WEBLOGIC_IMAGE_NAME_DEFAULT + ":" + imageTag;
-    getLogger().info(" The image name used for update is: {0}", imageUpdate);
+    getLogger().info(" The image name used for the 1st update is: {0}", imageUpdate);
+
+    //1st time patch domain resource with an image that does not exist in the registry so even with a new
+    //domain RestartVersion pods will not be restarted
+    patchDomainWithNewImage(imageUpdate);
+    String newRestartVersion = patchDomainResourceWithNewRestartVersion(domainUid, introDomainNamespace);
+    logger.log(Level.INFO, "New restart version is {0}", newRestartVersion);
+    //verify the server pods are not restarted because newImage is not available in the registry
+    //admin server pod will be in pending state(ImagePullBackOff)
```

Comment:
are we checking if the admin pod is in pending state here?

Reply:
In the test code we didn't

Reply:
I added code to explicitly verify before 2nd update admin server pod is in Pending state and restart count is 0